sysgen/sysgen.go

```diff
@@ -288,10 +288,13 @@ func generateArg(name, typ string, a []string, structs map[string]Struct, unname
 		}
 		fmt.Fprintf(out, "PtrType{%v, Dir: %v, Type: StrConstType{%v, Val: \"%v\"}}", common(), fmtDir("in"), common(), a[0]+"\\x00")
 	case "int8", "int16", "int32", "int64", "intptr":
-		if want := 0; len(a) != want {
-			failf("wrong number of arguments for %v arg %v, want %v, got %v", typ, name, want, len(a))
+		if len(a) == 2 {
+			range_begin := a[0]
```

Comment:
Go coding style is to use camelCase instead of c++_style. So:
s/range_begin/rangeBegin/